ansible-ipi-install/roles/installer/tasks/08_cleanup_sub_man_registeration.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: IPI-INSTALL Unregister host via subscription-manager
```

Comment:
why are we toggling subscription manger on/off during the installer steps, wouldn't this be in the node prep step?